pkg/ops/iso.go

```diff
@@ -31,10 +38,42 @@ func GenISO(name, src, dst string, i schema.ISO) func(ctx context.Context) error
 		}
 
 		log.Info().Msgf("Generating iso '%s' from '%s' to '%s'", name, src, dst)
-		out, err := utils.SH(fmt.Sprintf("/entrypoint.sh --debug --name %s build-iso --squash-no-compression --overlay-iso %s --date=false --output %s dir:%s", name, overlay, dst, src))
-		log.Printf("Output '%s'", out)
+		cfg, err := enkiconfig.ReadConfigBuild("/config", &pflag.FlagSet{})
```

Comment:
This code here seems to only load [this config](https://github.com/kairos-io/osbuilder/blob/405eda716a6c291eb539a98765a291e893d0eda1/tools-image/config.yaml). Given we hardcode the same setting [some lines below](https://github.com/kairos-io/AuroraBoot/pull/68/files#diff-8b2bb3456340e31af34769338d6fcf82e6eb8cbf9455501b35255138c4965943R53), can we skip this loading of external configuration?

We have a command with command line arguments. That should be enough to get user's input. I don't think we should accept configuration from all over, including hardcoded locations like this one.

Reply:
yes please, its time to clean this up

Reply:
done